### ISSUE TITLE
[New Rule] `vue/no-console`

### ISSUE THREAD

**Issue by wenfangdu** [vuejs/eslint-plugin-vue#2155]
**Please describe what the rule should do:**

Since Vue ^3.3 now [supports console in the template](https://github.com/vuejs/core/issues/7939), a `vue/no-console` rule is needed to regulate its usage.

`vue/no-console` should work exactly the same as [`no-console`](https://eslint.org/docs/latest/rules/no-console) but in the template.

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[x] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

With `'vue/no-console': 'error'`, the following code should throw:

```vue
<template>
  <button @click="console.log"></button>
</template>
```

**Comment by ota-meshi** [comment]
Thank you for the rule proposal!
That rule sounds good to me.

**Comment by wenfangdu** [comment]
@ota-meshi Dude, did you block me, I can't receive your messages.
